src/bundle/Resources/public/js/scripts/core/date.time.picker.js

```diff
@@ -104,7 +104,7 @@ class DateTimePicker {
         }
     }
 
-    onKeyUp(isMinute, event) {
+    onKeyUp(isHour, isMinute, isSecond, event) {
```

Comment:
as only one of these is used in single call, it can be made into single variable
```suggestion
    onKeyUp(type, event) {
```